frontend/apps/reader/modules/readergesture.lua

```diff
@@ -1300,6 +1304,10 @@ function ReaderGesture:gestureAction(action, ges)
         self.ui:handleEvent(Event:new("GotoNextChapter"))
     elseif action == "prev_chapter" then
         self.ui:handleEvent(Event:new("GotoPrevChapter"))
+    elseif action == "next_bookmark" then
+        self.ui:handleEvent(Event:new("GotoNextBookmarkFromPage", self:getCurrentPage()))
```

Comment:
Does this getCurrentPage() business have to really be in this readergesture file?
Can't we have just `self.ui:handleEvent(Event:new("GotoNextBookmarkFromCurrentPage"))` and have the getting of the current page in the 2 functions you added?